pkgs/development/python-modules/uamqp/default.nix

```diff
@@ -17,41 +17,19 @@
 
 buildPythonPackage rec {
   pname = "uamqp";
-  version = "1.6.9";
-  format = "setuptools";
+  version = "1.6.11";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "Azure";
     repo = "azure-uamqp-python";
     rev = "refs/tags/v${version}";
-    hash = "sha256-sDRIM41zey1F6/x1ZioJJBAQCVyf1NKzhS82Ew08pgM=";
+    hash = "sha256-HTIOHheCrvyI7DwA/UcUXk/fbesd29lvUvJ9TAeG3CE=";
   };
 
-  patches =
-    lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isx86_64) [
-      ./darwin-azure-c-shared-utility-corefoundation.patch
-    ]
-    ++ [
-      (fetchpatch {
-        name = "CVE-2024-25110.patch";
-        url = "https://github.com/Azure/azure-uamqp-c/commit/30865c9ccedaa32ddb036e87a8ebb52c3f18f695.patch";
-        stripLen = 1;
-        extraPrefix = "src/vendor/azure-uamqp-c/";
-        hash = "sha256-igzZqTLUUyuNcpCUbYHI4RXmWxg+7EC/yyD4DBurR2M=";
-      })
-      (fetchpatch {
-        name = "CVE-2024-27099.patch";
-        url = "https://github.com/Azure/azure-uamqp-c/commit/2ca42b6e4e098af2d17e487814a91d05f6ae4987.patch";
-        stripLen = 1;
-        extraPrefix = "src/vendor/azure-uamqp-c/";
-        # other files are just tests which aren't run from the python
-        # builder anyway
-        includes = [ "src/vendor/azure-uamqp-c/src/link.c" ];
-        hash = "sha256-EqDfG1xAz5CG8MssSSrz8Yrje5qwF8ri1Kdw+UUu5ms=";
-      })
-      # Fix incompatible function pointer conversion error with clang 16.
-      ./clang-fix-incompatible-function-pointer-conversion.patch
-    ];
+  patches = lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isx86_64) [
+    ./darwin-azure-c-shared-utility-corefoundation.patch
+  ];
```

Comment:
I don't have x86_64-darwin so can't test, but I wonder if this is still needed even after the Darwin rework. (It sucks that we have to wait for 9-10 hours for an ofborg eval.)

Reply:
I removed the patch as it didn't seem to be needed anymore. Thanks.